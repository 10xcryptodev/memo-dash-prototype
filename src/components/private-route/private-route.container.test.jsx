import React from 'react'
import configureStore from 'redux-mock-store'
import { shallow } from 'enzyme'
import PrivateRouteContainer from './private-route.container'
import { initialState } from '../../store/reducers/user.reducer'

describe('<PrivateRouteContainer />', () => {
  let store

  beforeEach(() => {
    // Mock store
    const mockStore = configureStore()
    store = mockStore({
      user: initialState,
      router: {}
    })

    const div = document.createElement('div')
    document.body.appendChild(div)
  })

  describe('Shallow rendering', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<PrivateRouteContainer />, { context: { store } })
      expect(wrapper).toMatchSnapshot()
    })
  })
})
