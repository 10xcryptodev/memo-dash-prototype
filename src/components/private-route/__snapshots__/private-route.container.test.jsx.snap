// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrivateRouteContainer /> Shallow rendering renders without crashing 1`] = `
<PrivateRouteComponent
  isLoggedIn={false}
/>
`;
